### PR TITLE
lint code in VREffect (Mr.doob™ approved)

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

**Comment by mrdoob** [comment]
Thanks!

File: examples/js/effects/VREffect.js [hunk 1]
```diff
@@ -9,7 +9,7 @@
  *
  */
 
-THREE.VREffect = function ( renderer, onError ) {
+THREE.VREffect = function( renderer, onError ) {
```
Comment (mrdoob) [inline examples/js/effects/VREffect.js]: @zz85 Is it possible to make https://zz85.github.io/mrdoobapproves/ add a space in anonymous functions?

```js
function ( par1, par2 ) {

	...

}
```

Comment (cvan) [inline examples/js/effects/VREffect.js]: yes! I prefer that syntax too 👍

Comment (cvan) [inline examples/js/effects/VREffect.js]: https://github.com/zz85/mrdoobapproves/pull/57/files

**Pull request merged**